packages/app/src/nativeBridge/modules/logsModule/index.ts

```diff
@@ -225,4 +248,9 @@ export class LogsModule extends NativeBridgeModule {
   public handleParserError(_mainWindow: BrowserWindow, _error: Error) {
     return;
   }
+
+  @moduleEvent('on')
+  public handleLogReadingTimeout(_mainWindow: BrowserWindow, _wowVersion: WowVersion, _timeoutSeconds: number) {
```

Comment:
do we need to pass timeoutSeconds in if that's a constant? would it be more meaningful to pass in the actual delta time?

Reply:
can swap to that, neither is strictly required for the callback to function correctly on the f/e side